src/main/java/it/uniroma2/pellegrini/z64sim/controller/SettingsController.java

```diff
@@ -204,6 +205,14 @@ private Settings() {
 
         protected static Settings loadConfiguration() throws SettingsException {
             Settings settings;
+            
+            // Following XDG standard if on *NIX-like
+            if(SystemUtils.IS_OS_LINUX || SystemUtils.IS_OS_UNIX) {
+                configurationDirectoryPath = System.getProperty("user.home") + "/.config/z64sim";
```

Comment:
Before this, we should check if `System.getenv("XDG_CONFIG_HOME")` is not null and use that in case. If null, we should fallback to `~/.config/`.